### PR TITLE
Allows varediting loc and coords of atom/movables

### PR DESCRIPTION
**Pull request by Cyberboss**
How the fuck has no one thought of this before?

### CODE REVIEW

File: code/game/atoms_movable.dm [hunk 1]
```diff
@@ -42,6 +42,30 @@
...
+				return TRUE
+			return FALSE
+		if("z")
+			var/turf/T = locate(x, y, var_value)
+			if(T)
+				forceMove(T)
+				return TRUE
+			return FALSE
+		if("loc")
+			if(istype(var_value, /atom))
```
Comment (MrStonedOne) [inline code/game/atoms_movable.dm]: so you can't intentionally put something into null space?

**Comment by lzimann** [comment]
Byond already does everything when you edit X/Y/Z afaik.

**Comment by MrPerson** [comment]
It doesn't call Move() or, in this case, forceMove().

**Comment by Cyberboss** [comment]
@lzimann Doesn't work for objects in containers

File: code/game/atoms_movable.dm [hunk 2]
```diff
@@ -62,7 +62,7 @@
 				return TRUE
 			return FALSE
 		if("loc")
-			if(istype(var_value, /atom))
+			if(var_value == null || istype(var_value, /atom))
```
Comment (RemieRichards) [inline code/game/atoms_movable.dm]: what if I want to send the atom to 27? HMM? HMMMM?

**Comment by MrStonedOne** [comment]
Call forceMove, but i'm not seeing any utility in trying to sanitize the values. If we start trying to sanitize VV values we will start creating too much reliance on that. Admins already know VV is a loaded gun, and make a good faith effort to take that into account, don't ruin that.

**Comment by PsiOmegaDelta** [comment]
For Bay I elected locating the turf at the resulting x,y,z location but with a min-pos of 1,1,1.
The reason is simply that this supposedly mimics BYOND's normal behaviour when objects inside other objects/nullspace have these vars changed and as such the goal was to maintain expectations if someone who knows the engine tinkers with these values.

**Pull request merged**